setup.sh

```diff
@@ -31,25 +31,49 @@ check_install_pkg() {
 
     fi
 }
+check_install_pip(){
+
+	pip_ver=$($pip --version | grep --line-buffered "^Version")
+	if [ -n "$pip_ver" ]; then
+		echo "$pip_ver found"
+
+	else
+		echo "pip not found, Installing... "
+        mkdir -p tools
```

Comment:
As this script assumes we're on some Ubuntu-flavored linux anyway (cf. line 75), I'd install pip / pip3 via apt here, i.e.
`sudo apt install python-pip` or `sudo apt install python3-pip`

Reply:
Used pip install, because it directly installs the latest version . 
However apt-get is faster as it installs pre-compiled binary and hence i updated the script to use apt ([here](https://github.com/direct-state-transfer/dst-doc/pull/15/commits/5a85463862306cdf8628ce1c0687fb20cc704600)).
if still the latest version is required then use `pip install --upgrade pip` to achieve it.